### PR TITLE
check_yum - exclude empty lines fix.

### PR DESCRIPTION
**Pull request by pbalu-code**
When number of updates is not matching with number of lines.

### CODE REVIEW

File: check_yum.py [hunk 1]
```diff
@@ -463,7 +463,8 @@ def get_security_updates(self):
              'Limiting package lists to security relevant ones',
              'This system is receiving updates from RHN Classic or Red Hat Satellite.',
              r'Repo [\w-]+ forced skip_if_unavailable=\w+ due to',
-             r'^\s*:\s+'
+             r'^\s*:\s+',
+             '^\s*'
```
[the remark this answers is not in the document]
Comment (HariSekhon) [reply check_yum.py:467]: you'd also need to be tighter on the regex to prevent it matching every single line, eg. `r'^\s*$'`

Comment (pbalu-code) [reply check_yum.py]: You are right but with "$"sign  it is not working. I have no idea why.

**Comment by HariSekhon** [comment]
Thanks for the PR!

**Comment by HariSekhon** [comment]
So it turns out this, by missing the end anchor, actually ignores all the unrecognized lines, that is why it suppresses the output format checks. I'll have to revert this, although I'll roll forwards because I'm doing some other updates too.

**Pull request merged**